labs/authentication-passport/src/__tests__/unit/passport-strategy-adapter.unit.ts

```diff
@@ -0,0 +1,75 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/authentication-passport
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+// FOR REVIWERS: THIS UNIT TEST IS FOR `StrategyAdapter`
```

Comment:
nitpick: `REVIWERS` -> `REVIEWERS`

Reply:
oh..another good catch, that comment should be moved too!